extensions.json

```diff
@@ -957,10 +991,11 @@
             "id": "watchonly",
             "repo": "https://github.com/lnbits/watchonly",
             "name": "Onchain Wallet",
-            "version": "0.2.2",
+            "version": "0.2.0",
```

Comment:
why revert version?

Reply:
I didnt, wondered if you did?

Reply:
fixed now

Reply:
https://github.com/lnbits/lnbits-extensions/pull/374/commits/32199084e1aa9e0325e8e683fe16e537f9fad0b3